sklearn/naive_bayes.py

```diff
@@ -484,13 +484,13 @@ def partial_fit(self, X, y, classes=None, sample_weight=None):
         y : array-like, shape = [n_samples]
             Target values.
 
-        classes : array-like, shape = [n_classes], optional (default=None)
+        classes : array-like, shape = [n_classes] (default=None)
```

Comment:
I see that we are not consistent here.

I think we should use tuples for shapes in doctrings so:

array-like, shape = (n_classes,) (default=None)